src/tests/Login.test.jsx

```diff
@@ -107,3 +107,23 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('input field', async () => {
+    render(<Login />)
+
+    fireEvent.change(screen.getByPlaceholderText('Username or Email'), {
+        target: { value: 'MyUsername' },
+    })
+    fireEvent.change(screen.getByPlaceholderText('Password'), {
+        target: { value: '12345678' },
+    })
+    expect(screen.queryByLabelText("errorMessage")).toBeNull();
+
+    act(() => {
+        fireEvent.click(screen.getByRole('button', { name: "Login" }), {
+            target: { value: 'true' },
+        })
+    });
+    expect(screen.queryByLabelText("errorMessage")).toBeNull();
```

Comment:
Here we want it to show warning on that field like the screenshot below
<img width="636" alt="Screen Shot 2020-09-01 at 11 44 59 am" src="https://user-images.githubusercontent.com/29667122/91785402-ad72be00-ec48-11ea-9733-3e8ac0b63c0d.png">